meetings/tool/src/main/resources/meeting-recordings_eu.properties

```diff
@@ -0,0 +1,5 @@
+back=Back
```

Comment:
@pablosanroman Could you provide these strings when you have time? Thanks

Reply:
Hi @mpellicer !

empty_field= Eremu hau nahitaez bete behar da
empty_checkbox= Kontrol-lauki hau nahitaez aktibatu behar da
maxlenght_field= Balioak gainditzen du baimendutako gehienezko tamaina

Salu:D!